tasks/evolutionary-algorithm/src/main.rs

```diff
@@ -4,94 +4,95 @@
 
 extern crate rand;
 
-use std::collections::HashMap;
 use rand::Rng;
 
 fn main() {
-    let target: String = String::from("METHINKS IT IS LIKE A WEASEL");
-    let mut parent: String = String::new();
-    let nb_copy = 400;
-    let mutation_rate: f64 = 0.05;
-    let mut counter = 0;
+    let target = "methinks it is like a weasel"; // lowercase for readability
```

Comment:
I don't like making this lower case, it seems like it's technically not following the task